sunflower-land/sunflower-land
# [FIX] Burn Wearables

# Description

This PR adds the ability to burn wearables inside of the game - this will be useful for players to upgrade their faction helmet into a crown.

We have a new Game contract. This PR also cleans up old unused functionality.

## How to test?

Update your `.env`:
`VITE_GAME_CONTRACT=0x6EC7018cA99bd820077D16724AAD72706524B1dF`

1. Connect to your API
2. Remove an item from your wardrobe
3. Sync on chain (you should see it burnt)

Also test a scenario of burning a supply limited item (e.g Wings). The backend will have extra cautions to prevent this from happening 👍
